src/main/kotlin/uk/gov/justice/digital/hmpps/externalmovementsapi/sync/TapApplicationRequest.kt

```diff
@@ -19,18 +16,9 @@ data class TapApplicationRequest(
   val movementApplicationId: Long,
   val eventSubType: String,
   val applicationDate: LocalDate,
-  val fromDate: LocalDate,
```

Comment:
Mike autogenerates his client from these so I'd be inclined to leave them in even though we're not doing anything with them